UnitTests/ReviewServiceTests.cs

```diff
@@ -0,0 +1,192 @@
+﻿using ConestogaInsidersClub.Data;
+using ConestogaInsidersClub.Data.DataAccess;
+using ConestogaInsidersClub.Data.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ServiceTests
+{
+    class ReviewServiceTests : TestBase
+    {
+        private Game testGame;
+        private Review testReview;
+        private List<ApplicationUser> users;
+
+        [OneTimeSetUp]
+        public async Task BeforeAll()
+        {
+
+            using var context = new ApplicationDbContext(ContextOptions);
+
+            testGame = new Game()
+            {
+                GameId = 1,
+                Name = "Age of Empires IV",
+                Description = "RTS Game"
+            };
+            await context.Set<Game>().AddAsync(testGame);
+
+            users = await SeedEntities(
+                new ApplicationUser()
+                {
+                    Id = "d123",
+                    UserName = "test",
+                    NormalizedUserName = "TEST",
+                    Email = "test@user.com"
+                },
+                new ApplicationUser()
+                {
+                    Id = "d124",
+                    UserName = "fake",
+                    NormalizedUserName = "FAKE",
+                    Email = "fake@user.com"
+                }
+            );
+
+            testReview = new Review()
+            {
+                GameId = testGame.GameId,
+                UserId = users[0].Id,
+                Rating = 5,
+                Description = "good game",
+                Approved = null
+            };
+            await context.Set<Review>().AddAsync(testReview);
+
+            await context.SaveChangesAsync();
+        }
+
+        [Test, Order(1)]
+        public async Task GetReviews()
+        {
+            //Arrange
+            using var context = new ApplicationDbContext(ContextOptions);
+            var service = new ReviewService(context);
+            //Act
+            var reviews = await service.GetReviewsAwaitingApproval();
+            //Assert
+            Assert.IsNotNull(reviews);
+            Assert.That(reviews, Has.Count.EqualTo(1));
```

Comment:
It could also be beneficial to check that all reviews are actually unapproved.

Reply:
I made a check to see if the review it grabs is actually a null review